src/env.cr

```diff
@@ -52,6 +47,31 @@ module ENV
     !!LibC.getenv(key)
   end
 
+  # Retrieve a value corresponding to a key. Raises a KeyError exception if the
+  # key does not exist.
+  #
+  def self.fetch(key)
+    fetch(key) do
+      raise KeyError.new "Missing ENV key: #{key.inspect}"
+    end
+  end
+
+  # Retrieve a value corresponding to a key. Return the second argument's value
+  # if the key does not exist.
+  #
+  def self.fetch(key, default)
+    fetch(key) { default }
+  end
+
+  # Retrieve a value corresponding to a key. Return the value of the block if
+  # the key does not exist.
+  #
```

Comment:
Mmmh, I haven't seen that doc style with the empty trailing comment seen anywhere else in the codebase ;)